includes/class-wc-siftscience-events.php

```diff
@@ -208,7 +208,7 @@ public function send_queued_data() {
 
 			foreach ( $this->events as $event ) {
 				if ( null != $event ) {
-					$this->comm->post_event( $event );
+					$this->_comm->post_event( $event );
 				}
 			}
 
```

Comment:
I think `events` and `order_map` should be declared and have underscores.

Reply:
there were declared way below wait next commit